### PR TITLE
Swap samtools depth to nftest

### PR DESCRIPTION
**Pull request by SPPearce**

### CODE REVIEW

File: modules/nf-core/samtools/depth/tests/main.nf.test [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(process.out).match() }
+            )
+        }
+
+    }
+
+    test("human - no_intervals - stub") {
```
Comment (maxulysse) [inline modules/nf-core/samtools/depth/tests/main.nf.test:61]: why not stub for with_intervals

File: modules/nf-core/samtools/depth/tests/nextflow.config
Comment (maxulysse) [inline modules/nf-core/samtools/depth/tests/nextflow.config:1]: I think you can delete this file

**Comment by maxulysse** [comment]
minor comments

**Comment by GallVp** [comment]
Sorry @SPPearce 

I did not realise I am duplicating it under #6275

**Comment by SPPearce** [comment]
Closed in favour of #6275

**Pull request closed without merging**